### PR TITLE
Use consistent camelCase in http api

### PR DESCRIPTION
**Pull request by ivarprudnikov**
Issue #187 

- rename json fields as per https://github.com/Azure/azure-rest-api-specs/pull/31896 
- remove dead code related to unused did:web resolver and signature issuance

### CODE REVIEW

File: app/src/kv_types.h [hunk 1]
```diff
@@ -156,33 +144,44 @@ namespace scitt
     Policy policy = {};
     Authentication authentication = {};
 
-    // The long-term stable identifier of this service, as a DID.
+    // The long-term stable issuer string of this service.
     // If set, it will be used to populate the issuer field of receipts
-    std::optional<std::string> service_identifier;
+    std::optional<std::string> service_issuer;
```
Comment (achamayou) [inline app/src/kv_types.h:149]: Same remark as above, `cose_signatures.issuer` is used for that purpose, not this value.

File: test/infra/fixtures.py [hunk 2]
```diff
@@ -306,16 +305,17 @@ def pytest_collection_modifyitems(config, items):
...
+    Get the long term service issuer string.
 
     The service is configured to include this identifier in receipts.
     """
-
     result = urlparse(service_url)
     assert result.hostname is not None
-    return format_did_web(result.hostname, result.port)
+    if result.port is None or result.port == 443 or result.port == 80:
+        return "https://" + result.hostname
```
Comment (achamayou) [inline test/infra/fixtures.py:317]: if result.port == 80 it's arguable that this should return "http://" + result.hostname instead.

File: app/constitution/scitt.js [hunk 3]
```diff
@@ -12,31 +12,25 @@ actions.set("set_scitt_configuration",
...
+        checkType(args.configuration.authentication.allowUnauthenticated, "boolean?", "configuration.authentication.allowUnauthenticated");
         checkType(args.configuration.authentication.jwt, "object?", "configuration.authentication.jwt");
         if (args.configuration.authentication.jwt) {
-          checkType(args.configuration.authentication.jwt.required_claims, "object?", "configuration.authentication.jwt.required_claims");
+          checkType(args.configuration.authentication.jwt.requiredClaims, "object?", "configuration.authentication.jwt.requiredClaims");
         }
       }
 
-      checkType(args.configuration.service_identifier, "string?", "configuration.service_identifier");
+      checkType(args.configuration.serviceIssuer, "string?", "configuration.serviceIssuer");
```
[the remark this answers is not in the document]
Comment (ivarprudnikov) [reply app/constitution/scitt.js:33]: left it but marked the property in KV as deprecated in the comment. It is still used in the did:web part of the code that generated the page but should be removed once we merge the transparency-config + jwks

File: app/src/kv_types.h (hunk 1 above)
Comment (ivarprudnikov) [reply app/src/kv_types.h:148]: add a section in the doc about `cose_signatures` and removed  `service_issuer` from python tests and from docs

File: test/infra/fixtures.py (hunk 2 above)
Comment (ivarprudnikov) [reply test/infra/fixtures.py]: this block was removed

**Pull request merged**